ArchiSteamFarm/ArchiWebHandler.cs

```diff
@@ -1701,6 +1701,30 @@ public sealed class ArchiWebHandler : IDisposable {
 			return result;
 		}
 
+		internal async Task<byte> GetCardCountForGame(uint appID) {
+			if (appID == 0) {
+				Bot.ArchiLogger.LogNullError(nameof(appID));
+
+				return 0;
+			}
+
+			using IDocument? htmlDocument = await GetGameCardsPage(appID).ConfigureAwait(false);
+
+			if (htmlDocument == null) {
+				return 0;
+			}
+
+			List<IElement> htmlNodes = htmlDocument.SelectNodes("//div[@class='badge_detail_tasks']//div[@class='badge_card_set_text ellipsis']");
```

Comment:
This code is directly from @JustArchi and used in @JustArchi-ArchiBot already. I think this should be fine

Reply:
If @Vital7 approach makes more sense for at least one reason and gives the same result, I'm all in for improving that part, just because I coded something that works doesn't mean it's the best solution.

Reply:
IMO it makes more sense since we need card count, so we count amount of cards divs, not text divs (amount of which is not reliable).

Reply:
IMO it makes more sense since we need card count, so we count amount of cards divs, not text divs (amount of which is not reliable).